src/components/Editor.vue

```diff
@@ -69,10 +69,10 @@
 import Vue, { set } from 'vue'
 import escapeHtml from 'escape-html'
 import moment from '@nextcloud/moment'
-import { getVersion, receiveTransaction } from 'prosemirror-collab'
-import { Step } from 'prosemirror-transform'
 import { getCurrentUser } from '@nextcloud/auth'
 import { loadState } from '@nextcloud/initial-state'
+import { Collaboration } from '@tiptap/extension-collaboration'
+import * as Y from 'yjs'
```

Comment:
Try to do not import `*` from any package.

If this package supports [tree shaking](https://webpack.js.org/guides/tree-shaking/), it will not be applied.